app/components/skill-list-item-link.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  get,
+  inject: { service },
+  set
+} = Ember;
+
+export default Component.extend({
+  classNames: 'skill-list-item-link',
+  classNameBindings: ['justClicked', 'justRemoved', 'matched'],
+  tagName: 'a',
+
+  session: service(),
+
+  click() {
+    if (get(this, 'session.isAuthenticated')) {
```

Comment:
Would this ever happen? I think you are only rendering this component if the session is authenticated as it is?